src/plugins/discover/public/application/main/components/sidebar/lib/visualize_trigger_utils.ts

```diff
@@ -14,6 +14,7 @@ import {
 } from '../../../../../../../ui_actions/public';
 import { getUiActions } from '../../../../../kibana_services';
 import { DataViewField, KBN_FIELD_TYPES } from '../../../../../../../data/common';
+import { APP_ID } from '../../../../../../common/index';
```

Comment:
This should also work:
```suggestion
import { APP_ID } from '../../../../../../common';
```